bridge-fungible-token/src/utils.sw

```diff
@@ -248,7 +283,70 @@ fn bn_mult(bn: U256, factor: u64) -> (U256, u64) {
 
         result: (U256, u64)
     };
-
     enable_panic_on_overflow();
     result
 }
+
+// TODO: [std-lib] replace when added as a method to U128/U256
+fn bn_div(bn: U256, d: u32) -> (U256, u32) {
+    let m: u64 = 4294967295;
```

Comment:
This might look cleaner if we denote it in hex format (0x00000000FFFFFFFF). It might make it more obvious that it's a bit mask to isolate the lower 32 bits.